cmd/wksctl/kubeconfig/kubeconfig.go

```diff
@@ -78,6 +79,14 @@ func kubeconfigRun(cmd *cobra.Command, args []string) error {
 	// TODO: deduplicate clusterPath/machinesPath evaluation between here and cmd/wksctl/apply
 	// https://github.com/weaveworks/wksctl/issues/58
 	if kubeconfigOptions.gitURL == "" {
+		// Check for cluster.yaml and machines.yaml paths
+		if _, err := os.Stat(kubeconfigOptions.clusterManifestPath); os.IsNotExist(err) {
+			kubeconfigOptions.clusterManifestPath = "./setup/cluster.yaml"
+		}
+
+		if _, err := os.Stat(kubeconfigOptions.machinesManifestPath); os.IsNotExist(err) {
+			kubeconfigOptions.machinesManifestPath = "./setup/machines.yaml"
+		}
```

Comment:
Tricky!
- I don't think we should "mix and match" here though in case the user has accidentally has:
  - `./cluster.yaml` and `./setup/machines.yaml` then weird things will happen.
- Also the error message here might be a bit funny if the user provides an override (`wk kubeconfig --machines=/tmp/machines.yaml` I think we might still print "./setup/machines.yaml" not found.?

Reply:
Thank you Simon. It makes a lot of sense 🙂 
I think we can check if the default path is set to root then we can apply these? Andd check for both files in the same dir..